src/ainft.ts

```diff
@@ -61,7 +61,7 @@ export default class AinftJs {
     const chainId = _.get(config, 'chainId') || AIN_BLOCKCHAIN_CHAIN_ID[stage];
     setEnv(stage);
 
-    this.ain = new Ain(endpoint, chainId);
+    this.ain = new Ain(endpoint, undefined, chainId);
```

Comment:
how about using 'null' explicitly here and below?

Reply:
@platfowner this constructor allows only string or undefined types.

Reply:
Ah, I added 'null' to the member variable but forgot updating the constructor prototype.. I'll do it soon. Please feel free to move forward.
https://github.com/ainblockchain/ain-js/blob/master/src/ain.ts#L27